content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -111,6 +111,59 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'planetscale']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Note that the default schema created by `prisma init` uses PostgreSQL as the `provider`. For PlanetScale, you need to use the `mysql` provider instead:
+
+```prisma file=prisma/schema.prisma highlight=2;edit
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+```
+
+In this case, the `url` is [set via an environment variable](/concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) which is defined in `.env`:
+
+```bash file=.env
+DATABASE_URL="mysql://janedoe:mypassword@planetscale-hostname/mydb?sslaccept=strict"
```

Comment:
This can probably look more like what you actually get from planetscale, e.g. `mysql://janedoe:mypassword@server.us-east-2.psdb.cloud/mydb?sslaccept=strict`

Reply:
Done.